thrift-parser.js

```diff
@@ -366,6 +410,34 @@ module.exports = (buffer, offset = 0) => {
     return result;
   };
 
+  const readUnion = () => {
+    let subject = readKeyword('union');
+    let name = readName();
+    let items = readUnionBlock();
```

Comment:
https://github.com/apache/thrift/blob/052abc39c60c54f9901993e76a81036b482ed6ba/compiler/cpp/src/thrift/parse/t_struct.h#L66

Since union members must be optional and unions are treated as `t_struct` as well (according to the [grammar](https://github.com/apache/thrift/blob/330b3f814a33ab8a740fca4da29106bd7e219b1d/compiler/cpp/src/thrift/thrifty.yy#L674)), I agree that union members should include the `option` which is explicitly `optional`.